core/App/ehsm_provider.cpp

```diff
@@ -938,7 +952,40 @@ ehsm_status_t GenerateQuote(ehsm_data_t *quote)
     return EH_OK;
 }
 
+/*
+ *  @brief check mr_signer and mr_enclave
+ *  @param quote quote data
+ *  @param mr_signer_good the mr_signer
+ *  @param mr_enclave_good the mr_enclave 
+ */
+bool verify_policy(uint8_t* quote, const char* mr_signer_good, const char* mr_enclave_good)
```

Comment:
actually, this verification is need to put inside the enclave.
so you need to add a new ecall to do this operation.

Reply:
Agree

Reply:
It will be move to enclave